tck/src/main/java/org/eclipse/microprofile/rest/client/tck/InvokeWithJsonPProviderTest.java

```diff
@@ -129,7 +132,10 @@ private void testGetSingle(JsonPClient client, String clientType) {
 
     private void testPost(JsonPClient client, String clientType) {
         reset();
-        stubFor(post(urlEqualTo("/")).willReturn(aResponse().withStatus(200)));
+        stubFor(post(urlEqualTo("/"))
```

Comment:
`Content-Type` should be immaterial to this test.

Reply:
See my comment above.